docker-local/library/prometheus/playbooks/templates/prometheus.yaml.tmpl

```diff
@@ -1,5 +1,5 @@
 global:
-  scrape_interval: 5s
+  scrape_interval: 15s
```

Comment:
15 seconds (merged version) was also the [scrape interval I think Mario set](https://github.com/SODALITE-EU/monitoring-system/blob/366f6d70e90ae7272f11a16b43318e1c0474eaeb/prometheus-config/prometheus.yml#L2) for the initial version in Y1, so I suppose 15s is OK.